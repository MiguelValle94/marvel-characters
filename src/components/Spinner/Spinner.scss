.spinner {
  &__wrapper {
    padding: 40px 0;
  }
}
