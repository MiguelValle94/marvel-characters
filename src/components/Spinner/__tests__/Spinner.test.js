import React from 'react';
import { shallow } from 'enzyme';

import Spinner from '../Spinner';

const setUpShallow = () => {
  const component = shallow(<Spinner />);
  return component;
};

describe('Spinner', () => {
  it('should match snapshot', () => {
    const component = setUpShallow();
    expect(component).toMatchSnapshot();
  });
});
