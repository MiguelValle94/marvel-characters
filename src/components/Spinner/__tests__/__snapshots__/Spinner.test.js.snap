// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner should match snapshot 1`] = `
<div
  className="spinner__wrapper"
>
  <div
    className="spinner-border"
    role="status"
  >
    <span
      className="sr-only"
    >
      Loading...
    </span>
  </div>
</div>
`;
